resource-timing/resources/buffer-full-utilities.js

```diff
@@ -1,13 +1,17 @@
 let appendScript = (src, resolve) => {
+    const po = new PerformanceObserver((list, observer) =>  { 
```

Comment:
Can you explain the purpose of this observer? It's not clear to me

Reply:
Its purpose is to make sure the script entry was added to the PerformanceObserver, under the assumption that this implies it was added to the performance timeline as well

Reply:
I'll add a comment to that effect